web/pwa/components/history/DaySummary.js

```diff
@@ -29,28 +32,56 @@ export function DaySummary({
     dayEnd
   );
 
+  const hasWorkMissions = React.useMemo(
+    () => missions.filter(mission => !mission.isHoliday).length > 0,
+    [missions]
+  );
+
   return (
     <>
-      <WorkTimeSummaryKpiGrid
-        loading={loading}
-        metrics={renderPeriodKpis(stats, true).filter(
-          m => m.name !== "workedDays"
-        )}
-      />
-      <InfoCard className={infoCardStyles.topMargin}>
-        {isDayEnded && activitiesWithNextAndPreviousDay.length > 0 ? (
-          <DayRegulatoryAlerts
-            day={isoFormatLocalDate(dayStart)}
-            userId={userId}
-            shouldDisplayInitialEmployeeVersion={
-              shouldDisplayInitialEmployeeVersion
-            }
-            prefetchedRegulationComputation={prefetchedRegulationComputation}
+      {hasWorkMissions && (
+        <>
+          <WorkTimeSummaryKpiGrid
+            loading={loading}
+            metrics={renderPeriodKpis(stats, true)
+              .filter(m => m.name !== "workedDays")
+              .filter(m => m.name !== "offDays")}
```

Comment:
:green_circle: `m` est souvent utilisé pour "mission", je propose un rename : 
```suggestion
              .filter(kpi => kpi.name !== "workedDays")
              .filter(kpi => kpi.name !== "offDays")}
```